### ISSUE TITLE
Question for accuracy metric in toy_example.py

### ISSUE THREAD

**Issue by qinhui99** [ilivans/tf-rnn-attention#3]
I found the codes of in toy_example.py likes these:
accuracy = 1. - tf.reduce_mean(tf.cast(tf.equal(tf.round(y_hat), target_ph), tf.float32))
I am confused.  Why don't you use the codes "accuracy = tf.reduce_mean(tf.cast(tf.equal(tf.round(y_hat), target_ph), tf.float32))" ?  Is it a bug ?

**Comment by ilivans** [comment]
This is a good point. Actually, both variants are wrong, because `y_hat` are logits rather than probabilities. It must be
`accuracy = tf.reduce_mean(tf.cast(tf.equal(tf.round(tf.sigmoid(y_hat)), target_ph), tf.float32))`
I will fix it shortly, thank you!

**Comment by ilivans** [comment]
Fixed
